packages/common/src/server.ts

```diff
@@ -508,6 +509,13 @@ export class YogaServer<
 
       return result
     } catch (error: unknown) {
+      if (error instanceof GraphQLYogaError) {
+        return getErrorResponse({
+          status: 200,
```

Comment:
I guess the status code here is debatable, any ideas or recommendations are welcome. I chose 200 because it is an expected error. 400 would mean that the request is invalid, I kind of feel 400 might be too strict.